changelog.d/13778.misc

```diff
@@ -0,0 +1 @@
+Fix GitHub done action skippable syntax.
```

Comment:
Ooops, meant to add in review: I think we'd normally copy the changelog from #13769 here: it makes the generated changelog at release time slightly nicer.